docs/guide/plugins/plugins.md

```diff
@@ -110,12 +110,18 @@ Currently the list contains the following functions:
 - `app.setBackgroundName` sets the name of the background layer
 - `app.scaleTextElements` scales all text elements of the current layer by the given scale factor
 - `app.getDisplayDpi` returns the configured display DPI
+- `app.export` exports the current document in pdf, svg or png format
+**(only available in [nightly release](https://github.com/xournalpp/xournalpp/releases/tag/nightly) as of v1.1.1)**
```

Comment:
To reduce the amount of work required to update these links, I would define a link alias at the bottom of the file:
```markdown
[nightly-release]: https://github.com/xournalpp/xournalpp/releases/tag/nightly)
```
which can then be used as follows: `only available in [nightly release][nightly-release]`

A longer-term solution is to define a macro in MkDocs that can pull these links out of the config.

Reply:
Thanks for the hint!